src/routes/playground/tags/playground/tags/result/tags/error-output/error-output.style.module.scss

```
@@ -0,0 +1,149 @@
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  opacity: 0.75;
+  margin-left: auto;
+}
+
+.message {
+  margin: 0;
+  padding: 0.75rem;
+  font-size: 0.95rem;
+  line-height: 1.55;
+  // Runtime errors carry their location on a second line; without this it runs
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)